docs/guides/sharing-the-browser-history.md

```diff
@@ -61,18 +61,18 @@ const myFeatureAppDefinition = {
 
     return {
       render: () => (
-        <Router history={historyService.history}>
+        <HistoryRouter history={historyService.history}>
           <App />
-        </Router>
+        </HistoryRouter>
       ),
     };
   },
 };
 ```
 
 The `history` property of the History Service is API-compatible with the history
```

Comment:
I think exposing a foreign API (history) that cannot be implemented completely and changes with the external dependency changes, wasn't a good idea. We should replace the API with a feature-hub API. LocationObject/ConsumerLocation was always a hassle and now has changed. History has changed although not completely supported.

Reply:
How do you want to support the [react-router use case](https://github.com/sinnerschrader/feature-hub/blob/864611d40ff51b6ece4c0e3d40827141bf39b001/docs/guides/sharing-the-browser-history.md#as-a-feature-app) then? Can you also elaborate what exactly the hassle was so that we can potentially improve on that?